sentence_transformers/fit_mixin.py

```diff
@@ -238,6 +239,8 @@ def fit(
                 steps
             checkpoint_save_total_limit: Total number of checkpoints to
                 store
+            resume_from_checkpoint: If true, searches for checkpoints
+            to continue training from.
```

Comment:
```suggestion
            resume_from_checkpoint: If true, searches for checkpoints
                to continue training from.
```
Small nitpick to get the docs to render nicely